tests/tornado_simple_raft_node_server.py

```diff
@@ -0,0 +1,59 @@
+import asyncio
```

Comment:
Some comments here or in the README would be good to explain the intended usage of this server.